### PR TITLE
Transfer pipelines to new drone

### PR DESCRIPTION
**Pull request by tasharnvb**
I couldn't fix the test pipeline so it has been removed.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,49 +1,36 @@
...
   build_docker_image:
-    image: docker:1.12
+    image: docker:17.09.0-ce
     environment:
-      - DOCKER_HOST=tcp://127.0.0.1:2375
+      - DOCKER_HOST=tcp://172.17.0.1:2375
     commands:
-      - docker build -t image .
-
-  test:
```
Comment (marcinc) [inline .drone.yml]: What happened with this step?

Comment (tasharnvb) [reply .drone.yml]: Connection refused: https://drone.acp.homeoffice.gov.uk/UKHomeOffice/docker-nginx/50/4

**Comment by vaijab** [comment]
Test pipeline is very important, otherwise what's the point of building something that's broken?

**Comment by tasharnvb** [comment]
Do you have any ideas on how to fix it?

**Comment by vaijab** [comment]
> Do you have any ideas on how to fix it?

Yep, the idea is to figure out what container IP is and curl it. Try below:
```
curl http://$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <container_name>)
```

**Comment by tasharnvb** [comment]
Closing this until I fix this issue with the test pipeline.

**Pull request closed without merging**